.github/workflows/windows-vcpkg.yml

```diff
@@ -170,7 +170,10 @@ jobs:
         batch-name: short-tests
         os: ${{ matrix.test-platform }}
 
-
+    - name: Solver archive creation
+      run: |
+           cd _build
+           tar czf solver.tar.gz solver/antares-*-solver solver/libsirius_solver.so
```

Comment:
1. Windows uses extension **.dll** for dynamic libs
2. You may need to specify `shell: bash`